modules/pub-sub/src/test/java/org/apache/ignite/stream/pubsub/PubSubStreamerSelfTest.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2018 Google LLC
```

Comment:
Do we need the copyright info or Apache 2.0 licence will suffice?

Reply:
Removed it and set the Apache 2.0 licence